### DISCUSSION TITLE
Welcome to criticalmaps-ios Discussions!

### DISCUSSION THREAD

**Discussion by mltbnz** [criticalmaps/criticalmaps-ios#463] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soraaerial** [comment]
Hi team. 

Have just stumbled across your product and it’s really impressive. I had the impression that it was loosely linked to criticalmass.in which appears to have been officially deprecated at this point. 

With this in mind, there is no way for new cities to add rides or schedules for new rides. Is this a feature in the app or website that already exists or is this something that will be added?

I noticed in the app (iOS) there are notification options for different ride categories, but as a ride organiser, I can’t see how these are added. 

Thanks!

**Reply by mltbnz** [reply]
Hi.
Thanks for bringing this to the attention. I didn't know it was deprecated. Bummer

The next event that will be shown on the map is fetched from criticalmass.in
I would love to continue supporting this feature but currently nothing is planed to add this functionality to our website or similar.

I think it would need some work since it would require users to register I guess.

But I'll raise the issue in our slack and see if we could provide a solution

**Reply by soraaerial** [reply]
Thank you. Yes it is quite confusing as the site is still up. I haven’t been able to add a new ride let alone a new city sadly.

**Reply by mltbnz** [reply]
Seems like the API is still running but it's of course sad users can't add anything.

Tbh I don't think this will be added to this app or our website anytime soon although I'd love to have that feature.

Do you write code and want to give it a go maybe?

**Reply by mltbnz** [reply]
Maybe we should adjust the banner text in app now that users can't add new rides to it

**Comment by genevievecory** [comment]
criticalmass.in posted here that they were going to be shutting down at the end of September 2022 - https://criticalmass.blog/nach-elf-jahren-ist-schluss-criticalmass-in-sperrt-zu/ - but that appears to have been averted?

**Reply by mltbnz** [reply]
Thanks. I've read the post and was asking myself the same question.

We are currently trying to become a Verein which I guess could also be an option. I'll check with Malte
